templates/include/glean/glean_base.1.schema.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": {
+    "type": "string",
+    "enum": [
+      "moz://mozilla.org/schemas/glean/ping/1"
+    ]
+  },
+  "client_info": @GLEAN_CLIENT_INFO_1_JSON@,
+  "ping_info": {
```

Comment:
I'd have expected the `*_info` sections to only be present in the normal schema.  Is there a reason the `*_info` sections are still being included in the min schema?

Reply:
My thinking was that info sections could be included in the min schema but were not a requirement. Happy to change that though. Minimum schema here just relaxed the requirements while keeping the schema the same.

Reply:
Given the current planned implementation of not sending any `*_info` data for such pings, meaning all of those fields would always be null, I think it would help avoid confusion if the fields were omitted from the min schema (and thus the BQ tables/views).

However there may well be other considerations/trade-offs, like if it'd be a prohibitive amount of work to update all the places existing code assumes certain `*_info` fields exist for all pings (e.g. `client_info.client_id`), or perhaps planning ahead for when we might send some of the `*_info` data in the future and cherry picking individual fields for the schema would be too complicated.

Reply:
> if it'd be a prohibitive amount of work to update all the places existing code assumes certain `*_info` fields exist for all pings (e.g. `client_info.client_id`)

@scholtzan I think you'd know best about this.  What do you think?

Reply:
I do agree that a simpler bq schema would be good and way less confusing. I could really go either way on this, however given that the option for turning this off from the SDK is a binary on/off it probably makes the most sense to just remove the info sections from the minimum glean schema.

Reply:
Yeah, if we are not planning to collect that data and it's not possible from the SDK to even send it for these minimal pings, then it's best to leave it away

Reply:
Updated.